prefix_copy_gcc_libs: try and get latest libs copied last

Try to get the most recent GCC libs copied to the shared place.  We need
the latest because they should be backwards compatible, thus also work
with older binaries.

--- gcc-config-2.7/gcc-config
+++ gcc-config-2.7/gcc-config
@@ -619,8 +619,9 @@
 		rmdir "${sourcedir}"
 	}
 
-	# We don't rely on iteration order here.
-	local GCC_PROFILES=$(LC_ALL="C" ls ${GCC_ENV_D}/${CHOST}-*)
+	# We want the latest version to provide the final link to an object,
+	# for gcc's libs should be backwards compatible
+	local GCC_PROFILES=$(version_sorted_paths ${GCC_ENV_D}/${CHOST}-*)
 
 	local targetdirs= GCC_PATH= LDPATH=
 	for x in ${GCC_PROFILES} ; do
